.github/workflows/nightly.yml

```
@@ -0,0 +1,43 @@
+
+name: Deploy Nightly
+on:
+  # This can be used to automatically publish nightlies at UTC nighttime
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+  # This can be used to allow manually triggering nightlies from the web interface
+  workflow_dispatch:
+
+jobs:
+  wled_build:
+    uses: ./.github/workflows/build.yml
```

actionlint (1.7.10)

could not read reusable workflow file for "./.github/workflows/build.yml": open /home/jailuser/git/.github/workflows/build.yml: no such file or directory

(workflow-call)